nrf-hal-common/src/monotonic.rs

```diff
@@ -1,13 +1,46 @@
 /*!
 Implements the [Monotonic](rtic_monotonic::Monotonic) trait for the TIMERs and the RTCs.
 
-<Här borde vi ha användings exempel>
+<Strong> Example (RTC): </Strong>
+```rust
+// RTC0 with a frequency of 32 768 Hz
+type MyMono = MonotonicRtc<RTC0, 32_768>;
 
-< Här borde vi ha docs om RTC>
+// Make sure lfclk is started
+let clocks = hal::clocks::Clocks::new(cx.device.CLOCK);
+let clocks = clocks.start_lfclk();
+
+let mono = MyMono::new(cx.device.RTC0, &clocks).unwrap();
+```
+
+<Strong> Example (Timer): </Strong>
+```rust
+// TIMER0 with a frequency of 16 000 000 Hz
+type MyMono = MonotonicTimer<TIMER0, 16_000_000>;
+let mono = MyMono::new(cx.device.TIMER0);
+
+```
+
+A simple example using the timer/rtc can be found under `nrf-hal/examples/monotonic-blinky`
+
+### RTC - Real-time counter
+The [`rtc`] [§6.22](https://infocenter.nordicsemi.com/pdf/nRF52840_PS_v1.7.pdf#page=367)
+
+The prescaler is 12 bit wide, (0 <= prescaler <= 4095). The frequency can be calculated by:
```

Comment:
I think that this should be reworded in the future to make it read more fluently.